testsuites/syncgateway/functional/tests/test_log_rotation.py

```diff
@@ -41,14 +62,18 @@ def test_log_rotation_default_values(params_from_base_test_setup, sg_conf_name):
     cluster_hosts = cluster_helper.get_cluster_topology(cluster_conf)
     sg_one_url = cluster_hosts["sync_gateways"][0]["public"]
     sg_helper.stop_sync_gateway(cluster_config=cluster_conf, url=sg_one_url)
+
     # read sample sg_conf
-    with open(sg_conf) as defaul_conf:
-        data = json.load(defaul_conf)
+    server_url = utils.host_for_url(cluster_hosts["couchbase_servers"][0])
```

Comment:
Per the format ` http:[//[user:password@]host[:port]][/]path[?query][#fragment]` for a url, I am hoping to future proof this utility function so that it can do the following,

http://192.168.33.10:8091 would return 192.168.33.10 and
http://somebox.com:8091 would return somebox.com